addons/website_forum/populate/forum_post.py

```diff
@@ -17,7 +17,13 @@ class Post(models.Model):
 
     def _populate_factories(self):
         forum_ids = self.env.registry.populated_models['forum.forum']
-        hours = [_ for _ in range(1, 49)]
+        tag_ids = {
+            tag[0].id: tag[1] for tag in self.env["forum.tag"]._read_group(
+                [('forum_id', 'in', forum_ids)],
+                ['forum_id'],
+                ['id:array_agg'],
+        )}
```

Comment:
Oula, it's been a while since I messed with populate, can you remind me what this is doing? 😄